### PR TITLE
generate installer payload dynamically from ocp installer version

### PR DESCRIPTION
**Pull request by shylesh**
Signed-off-by: Shylesh Kumar Mohan <shmohan@redhat.com>

### CODE REVIEW

File: ocs_ci/deployment/flexy.py [hunk 1]
```diff
@@ -88,8 +88,28 @@ def deploy_prereq(self):
...
+        If 'nigtly' is present then we will use registry.svc to get latest
+        nightly else if '-ga' is present then we will look for
+        ENV_DATA['installer_payload_image']
+
+        """
+        payload_img = {"installer_payload_image": None}
+        vers = version or config.DEPLOYMENT['installer_version']
+        installer_version = expose_ocp_version(vers)
+        payload_img['installer_payload_image'] = (
+            ":".join([constants.REGISTRY_SVC, installer_version])
```
Comment (petr-balogh) [inline ocs_ci/deployment/flexy.py:109]: you could just do str formationg like f"{constants.REGISTRY_SVC}:{installer_version}"

**Review by petr-balogh — APPROVED** [review]

**Review by vavuthu — APPROVED** [review]

File: ocs_ci/deployment/flexy.py (hunk 1 above)
Comment (shylesh) [reply ocs_ci/deployment/flexy.py:109]: Agree, that would been simpler. Since I have approvals m keeping the same. Let me know if you need the change.

**Comment by shylesh** [comment]
Verified locally,


```html
Build downloaded                                                             version provided
----------------------------                                                      ----------------------

4.4.0-0.nightly-2020-08-23-194129                                                   --ocp-version 4.4 


registry.svc.ci.openshift.org/ocp/release:4.4.17"                               --ocp-version 4.4-ga

 "registry.svc.ci.openshift.org/ocp/release:4.5.7"                                 --ocp-version 4.5-ga

registry.svc.ci.openshift.org/ocp/release:4.5.0-0.nightly-2020-08-24-135141"         --ocp-version 4.5

File: ocs_ci/deployment/flexy.py (hunk 1 above)
Comment (petr-balogh) [reply ocs_ci/deployment/flexy.py:109]: No need for change ;) just tip for next time ;)

**Pull request merged**